### PR TITLE
HM_jdbc2

### PR DESCRIPTION
**Pull request by DadizhaO**

### CODE REVIEW

**Review by Daniluko — APPROVED** [review]
Good

File: src/main/java/Lesson15/OfficeDaoImpl.java [hunk 1]
```diff
@@ -43,4 +45,54 @@
...
+        String sql = "insert into offices(office,city,region,sales) values (?,?,?,?)";
+        PreparedStatement state = connection.prepareStatement(sql);
+        state.setBigDecimal(1, office.getOfficeId());
+        state.setString(2, office.getCity());
+        state.setString(3, office.getRegion());
+        state.setBigDecimal(4, office.getSales());
+
+        boolean update = state.executeUpdate() > 0;
+
+        state.close();
```
Comment (SerhiiPasko) [inline src/main/java/Lesson15/OfficeDaoImpl.java]: close it in finally block. Becouse if exception ...

File: src/test/java/Lesson15/OfficeDaoImplIntegrationTest.java [hunk 2]
```diff
@@ -24,4 +34,23 @@ public void testGetByRegion() throws SQLException {
...
+    public void test2UpdateOffice() throws SQLException {
+        assertTrue(officeDao.updateOffice(office2));
+
+    }
+
+    @Test
+    public void test3DeleteOffice() throws SQLException {
+        assertTrue(officeDao.deleteOffice(office2.getOfficeId()));
+        Set<Office> offices = officeDao.getAllOffices();
+        assertFalse(offices.contains(office1));
```
Comment (SerhiiPasko) [inline src/test/java/Lesson15/OfficeDaoImplIntegrationTest.java:53]: Your office1 and office2 there are same object in DB. Your is possible, but may hard understand

**Review by SerhiiPasko — CHANGES_REQUESTED** [review]
Read comment

**Review by SerhiiPasko — APPROVED** [review]
Good

**Pull request merged**